src/java/org/apache/cassandra/db/memtable/TrieMemtable.java

```diff
@@ -546,6 +562,16 @@ public int size()
             return data.valuesCount();
         }
 
+        public long numberOfRows()
+        {
+            long total = 0;
+            for (BTreePartitionData v: data.values())
+            {
+                total += v.tree.length;
+            }
+            return total;
+        }
```

Comment:
Nit: I see this method is added, but not used. Do we need it? One reason I ask is that the SonarCloud Quality Gate failed because this PR does not have sufficient test coverage and this code block is likely why.